jazz-build-module/apigee/Proxygen_template.properties

```diff
@@ -0,0 +1,32 @@
+#Swagger version 2.0 file location
+swagger.File.Name={{SwaggerFile}}
+
+#Proxy Endpoint details
+proxy.name={{ProxyName}}
+proxy.description={{ProxyDescription}}
+proxy.version=v1
+
+#Target Endpoint Details
+target.name=aws
+target.service.name=lambda
+target.description={{TargetDescription}}
+target.service.URI=/v1/common
+
+
+#below properties does not require a change for each build
+#Recomended to set the value to true
+proxy.overwritefiles=true
+
+#Recomended to set the value to true
+target.overwritefiles=true
+
+#Proxy Virtual Host comma seperated values(don't change this value unless if there is a requirement to change)
+proxy.virtualHostList=secure
+
+#Proxy end point Properties.
+code.template.rootFolder=./Templates
```

Comment:
As a convention, can we use small case for folder & file names ? 

./Templates => ./templates

Proxygen_template.properties => proxygen_template.properties